### PR TITLE
Added cursor to Staff Filing btn

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#7759

*Description of changes:*

* Added pointer to Staff Filing activator on hover.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/StaffNotation.vue [hunk 1]
```diff
@@ -119,6 +119,10 @@ export default class StaffNotation extends Vue {
     font-size: 0.875rem;
     color: $app-blue;
     border-right: 1px solid $gray3;
+
+    &:hover {
```
Comment (severinbeauvais) [inline src/components/Dashboard/StaffNotation.vue:123]: Just out of curiousity, what do you get if you don't specify the hover state?

**Review by severinbeauvais — APPROVED** [review]

File: src/components/Dashboard/StaffNotation.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/Dashboard/StaffNotation.vue:123]: No difference in this case!

Comment (severinbeauvais) [reply src/components/Dashboard/StaffNotation.vue:123]: Oh right, the hover state would be to change the look of this "button"... not used in this case.

Comment (cameron-freshworks) [reply src/components/Dashboard/StaffNotation.vue:123]: Totally, I guess i did that out of habit 😁

**Pull request merged**